CAM/HardAttachCamToPedBone.md

```diff
@@ -0,0 +1,40 @@
+---
+ns: CAM
+aliases: ["0x149916F50C34A40D","_ATTACH_CAM_TO_PED_BONE_2"]
+---
+## HARD_ATTACH_CAM_TO_PED_BONE
+
+```c
+// 0x149916F50C34A40D
+void HARD_ATTACH_CAM_TO_PED_BONE(Cam cam, Ped ped, int boneIndex, float xRot, float yRot, float zRot, float xOffset, float yOffset, float zOffset, BOOL isRelative);
+```
+
```

Comment:
Here is a description of the native based on the wiki of vhub
"Attaches a camera to a specific bone of a Ped, including full matrix transformations for both rotation and position offsets."